Realm/ObjectServerTests/SwiftPermissionsAPITests.swift

```diff
@@ -204,18 +204,18 @@ class SwiftPermissionsAPITests: SwiftSyncTestCase {
             XCTAssertNil(error)
             ex2.fulfill()
         }
-        waitForExpectations(timeout: 2.0, handler: nil)
+        waitForExpectations(timeout: 2000.0, handler: nil)
```

Comment:
Why is this change necessary?

Reply:
I accidentally left it in. The next commit I push will change it back to 2 seconds.